Support for keyword "name" in template key bindings (#2129)
latex3/latex2e: base/doc/ltnews44.tex

%
\taggingissue[s]{1402 and 1415 among others}

\section{Code improvements}

\subsection{Support for \cs{UseName} in template key bindings}

We have extended the logic in the key-binding argument for
\cs{DeclareTemplateCode}, such that you can use \cs{UseName} here without any

> performance impact when using templates. This allows automatic key and variable

Reviewer: ```suggestion
performance impact when using templates. This allows automatic variable
```

keys can already use - and numerals without any issue, it is just the variables which are a problem.

creation with \enquote{awkward} characters, most notably \texttt{-} and

> numerals.

Reviewer: ```suggestion
numerals. Of course, such variables then also need to be referred to via \cs{UseName} or some equivalent method in the code, so this is only useful if there is a pressing need for such special names.
```

%
\taggingissue[s]{1402 and 1415 among others}

\section{Code improvements}

> \subsection{Support for \cs{UseName} in template key bindings}

Reviewer: ```suggestion
\subsection{Support the keyword \texttt{name} in template key bindings}
```

or some similar text

We have extended the logic in the key-binding argument for

> \cs{DeclareTemplateCode}, such that you can use \cs{UseName} here without any

Reviewer: ```suggestion
\cs{DeclareTemplateCode}, such that you can use the keyword \texttt{name} here (in addition to \texttt{global})  without any
```
